boilerplate/openshift/golang_osd_hive_operator/includes.mk

```diff
@@ -0,0 +1,4 @@
+# This is a rollup of includes in this convention, for easier inclusion
+# by the repo's Makefile, which should just include this file.
+include boilerplate/openshift/golang_osd_cluster_operator/project.mk
```

Comment:
Should be `openshift/golang_osd_hive_operator` right?

Reply:
Yeah.

This made me go off and implement #28, in which this file goes away.

It was only when I was done with it that I realized I would need to address this issue anyway, because the path is also hardcoded into standard.mk. So

/hold

What I'll actually want to do is have standard.mk discover the path and set a variable, in a fashion similar to what #25 did. That way it'll work wherever we drop it.